Playground/GeneratePlayground.swift

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/swift
+import Foundation
+
+extension FileManager {
+    func directoryExists(at url: URL) -> Bool {
+        var isDirectory: ObjCBool = false
+        return fileExists(atPath: url.path, isDirectory: &isDirectory) && isDirectory.boolValue
+    }
+}
+
+let fileManager = FileManager()
+let isDebuggable: Bool = UserDefaults.standard.bool(forKey: "debug") // use -debug
```

Comment:
Setting this to `true` means that the playground would be generated with the **SwiftShortcuts.playgroundmodule** included in the **UserModules** directory so that all the source files are accessible from inside the Swift Playgrounds app.

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/482871/86541871-744bf500-bf08-11ea-8562-f1fd8477f201.png">

Useful if you like to maintain the project on the go 🙂 (but a pain to get code back into Git)

Reply:
I love this `debug` flag and the user of `UserDefaults` to get the argument values. Maybe we can remove the explicit `Bool` type specification since it can be inferred from the return value of `bool(forKey:)`?

```suggestion
let isDebuggable = UserDefaults.standard.bool(forKey: "debug") // use -debug
```